src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/BuildUpToDateCheck.cs

```diff
@@ -118,12 +120,27 @@ private void OnProjectChanged(IProjectSubscriptionUpdate e)
             _isDisabled = disableFastUpToDateCheckString != null && string.Equals(disableFastUpToDateCheckString, TrueValue, StringComparison.OrdinalIgnoreCase);
 
             _msBuildProjectFullPath = e.CurrentState.GetPropertyOrDefault(ConfigurationGeneral.SchemaName, ConfigurationGeneral.MSBuildProjectFullPathProperty, _msBuildProjectFullPath);
-            foreach (var referenceSchema in ReferenceSchemas)
+            _markerFile = e.CurrentState.GetPropertyOrDefault(ConfigurationGeneral.SchemaName, ConfigurationGeneral.CopyUpToDateMarkerProperty, null);
```

Comment:
Unfortunately, we still need CheckMarkers. The markers have to be checked independently of the project's inputs/outputs because they are independent of one another (i.e. a dependent assembly might have an implementation-only change that won't cause the outputs of the project to be rebuilt).